src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -42,7 +42,7 @@ public UserService(@Qualifier("userRepository") UserRepository userRepository) {
 
   public User createUser(User newUser) {
     newUser.setToken(UUID.randomUUID().toString());
-    newUser.setStatus(UserStatus.OFFLINE);
+    newUser.setStatus(UserStatus.ONLINE);
```

Comment:
are you sure that you want it as an enum and not as a boolean?

Reply:
And I understand that it makes it easier if the status is online off the bat but technically i don't think it makes much sense. Yes the odds that I am able to create a user and then not able to log in with the newly created user are extremely slim but still I think creation per se should not be online.

Reply:
> are you sure that you want it as an enum and not as a boolean?

Yes pretty sure: We can extend the status in the future easily (to BUSY, TYPING, and so on)



> And I understand that it makes it easier if the status is online off the bat but technically i don't think it makes much sense. Yes the odds that I am able to create a user and then not able to log in with the newly created user are extremely slim but still I think creation per se should not be online.

You are right, the frontend shall make another request for loggin in after registration.